misc/shtool

```diff
@@ -267,7 +267,7 @@ eval `echo h.$opt_spec |\
 eval `echo h:help,$opt_alias |\
       sed -e 's/-/_/g' -e 's/\([a-zA-Z0-9]\):\([^,]*\),*/opt_ALIAS_\2=\1;/g'`
 
-#   interate over argument line
+#   iterate over argument line
```

Comment:
This is a generated file from an upstream project called [GNU shtool](https://www.gnu.org/software/shtool/shtool.html). We're using an older version, but the latest release still has these typos. Given that the newest release is from 2008, I don't imagine this will get fixed upstream.